go/vt/vitessdriver/rows_test.go

```diff
@@ -135,3 +137,90 @@ func TestRows(t *testing.T) {
 
 	_ = ri.Close()
 }
+
+func TestColumnTypeScanType(t *testing.T) {
+	var r = sqltypes.Result{
+		Fields: []*querypb.Field{
+			{
```

Comment:
Can you please add a comment that describe what the expectations of the test is? This falls under our review item:

> If a test is added or modified, there should be a documentation on top of the test to explain what the expected behavior is what the test does.

Reply:
Done!